### PR TITLE
SPLAT-984: blocked-edges/4.13.0-*: Declare `NutanixCCMNeedsConfigMap`

### PR DESCRIPTION
**Pull request by petr-muller**
Generated by writing the rc.0 declaration by hand, and then copying out to other 4.13 prereleases with:

```
$ curl -s 'https://api.openshift.com/api/upgrades_info/graph?channel=candidate-4.13' | \
    jq -r '.nodes[].version' | \
    grep '^4[.]13[.]' | \
    grep -v '^4[.]13[.]0-rc[.]0$' | while read V;
        do sed "s/4[.]13[.]0-rc[.]0/${V}/g" 4.13.0-rc.0-NutanixCCMNeedsConfigMap.yaml  > "${V}-NutanixCCMNeedsConfigMap.yaml";
    done
```

I am not sure how we eventually limit this, though. This will be resolved by 4.12 having an `Upgradeable=false` in some version (until the user provides the CM). We would bump the minimum version in 4.12 to have this fix, but we would still need to carry the conditional risk for all future 4.13 versions? Because this will never have a `fixedIn` version in 4.13. Not sure.

### CODE REVIEW

File: blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml
Comment (LalatenduMohanty) [inline blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml:3]: I am not comfortable with linking the impact assessment jira card when the answers are not present. We can use the bug link here till the impact assessment card has more details.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml
Comment (wking) [reply blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml:3]: or we can copy over whatever the relevant bits from the OCPBUGS ticket are?  The impact-statement ticket is mutable, and anything we want to say, we can say there.

**Comment by sdodson** [comment]
/hold
When will we have this in 4.12? I'd prefer we just get that done given the considerations Petr mentions.

**Comment by sdodson** [comment]
/hold cancel
We won't have to declare this indefinitely, only on the limited set of 4.13.0-rc.0 before we land this in 4.12 and raise the 4.12.z bar for upgrades to 4.13.

**Comment by petr-muller** [comment]
/hold

I want to add PromQL to check for the CM presence and also address Lala's comment.

**Comment by petr-muller** [comment]
I have added PromQL that queries the absence/presence of the `cloud-conf` ConfigMap in `openshift-cloud-controller-manager`.

File: blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml
Comment (petr-muller) [reply blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml:3]: It seems the current consensus is to wait for this one - if we decide we want to merge, we'll want to update SPLAT-984. I'm not doing that for now.

**Comment by petr-muller** [comment]
/close

It is unlikely that we would need this

**Pull request closed without merging**
